Tests/SwiftLexicalScopesTest/SimpleQueryTests.swift

```diff
@@ -0,0 +1,131 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import SwiftLexicalScopes
+import XCTest
+
+final class testSimpleQueries: XCTestCase {
+  func testLabeledStmtLookupThreeNestedScopes() {
+    assertLexicalScopeQuery(
+      source: """
+        while true {
+          5️⃣break
+        }
+
+        1️⃣one: for i in 1..<10 {
+          while true {
+            2️⃣two: do {
+              3️⃣break one
+            } while true
+          }
+          4️⃣break
+        }
+        """,
+      methodUnderTest: { argument in
+        LexicalScopes.lookupLabeledStmts(at: argument)
+      },
+      expected: ["3️⃣": ["2️⃣", "1️⃣"], "4️⃣": ["1️⃣"], "5️⃣": []]
```

Comment:
The function is supposed to return all the labeled statements accessible at the position of a given node so in the case of the while, which is unlabeled, there aren’t available any labeled statements. The function just uses the `break` location to start lookup. Maybe it was a bit confusing to put unlabeled `break` in the test?

Reply:
Ah, I see, makes sense. I just thought that it would return the location that the `break` would jump to or something like that. But I might also just not have read the test properly.